### PR TITLE
when installing sequitur and mmseg, look for lib64 as well

### PR DESCRIPTION
**Pull request by jtrmal**
should be addressing #1672

### CODE REVIEW

File: tools/extras/install_mmseg.sh [hunk 1]
```diff
@@ -39,29 +39,47 @@ if [ -d ./mmseg-1.3.0 ] ; then
...
+)
 
-  [ -f ./env.sh ] && . ./env.sh
+## we first find the mmseg.py file (the module name which will be imported,
+## so that should be pretty reliable) and then we work out the location of
+## the site-packages directory (typically it would be one level up from
+## the location of the mmseg.py file but using find seems more reliable
+mmseg_file_lib=$(find ./mmseg-1.3.0/lib/ -type f -name mmseg.py | head -n 1)
+mmseg_file_lib64=$(find ./mmseg-1.3.0/lib64/ -type f -name mmseg.py | head -n 1)
+if [ -f $mmseg_file_lib ]; then
```
Comment (akshayc11) [inline tools/extras/install_mmseg.sh]: For some reason, this check is failing on my machine: The $mmseg_file_lib variable is empty, but despite that, this condition is being triggered.

How about this:

```
if [ "x${mmseg_file_lib}" != "x" ]; then
  lib_dir=./lib/
elif [ "x${mmseg_file_lib64}" != "x" ]; then
  lib_dir=./lib64
fi
```

The above worked for me, although, as far as I am aware, there seems to be no reason why the one you have proposed should not work

File: tools/extras/install_mmseg.sh [hunk 2]
```diff
@@ -39,29 +39,47 @@ if [ -d ./mmseg-1.3.0 ] ; then
...
+else
+  echo >&2 "$0: ERROR: Didn't find ./mmseg-1.3.0/lib/ or ./mmseg-1.3.0/lib64/"
+  echo >&2 "  Perhaps your python or system installs python into a different dir?"
+  echo >&2 "  or some other unknown issues arised - review the script output"
+  echo >&2 "  and try to figure out what went wrong."
+  exit 1
+fi
 
-  echo "export PYTHONPATH=\$PYTHONPATH:$wd/mmseg-1.3.0/lib/python${pyver}/site-packages"
+site_packages_dir=$(cd ./mmseg-1.3.0; find $libdir -name "site-packages" -type d | head -n1)
```
Comment (akshayc11) [inline tools/extras/install_mmseg.sh]: Should be 
```
+site_packages_dir=$(cd ./mmseg-1.3.0; find $lib_dir -name "site-packages" -type d | head -n1)
```

File: tools/extras/install_mmseg.sh (hunk 1 above)
Comment (akshayc11) [inline tools/extras/install_mmseg.sh]: For some reason this was failing in my code. I tried the following:
```
if [ ! -z ${mmseg_file_lib+x} ]; then
  lib_dir=./lib/
elif [ ! -z ${mmseg_file_lib64+x} ]; then
  lib_dir=./lib64/
else
...
```
Based on https://stackoverflow.com/questions/3601515/how-to-check-if-a-variable-is-set-in-bash

**Comment by akshayc11** [comment]
There was 1 typo, and 1 correction (using -z instead of -f) in the install_mmseg.sh. Please refer to the inline comments for more details.. Otherwise, worked in my machine. Looks good to me.

**Comment by jtrmal** [comment]
Thanks for testing it. Now those issues should be addressed.
Just to be sure: did you test the sequitur install as well?
y.

**Comment by akshayc11** [comment]
I did... Sequitur passed.

**Comment by danpovey** [comment]
ok to merge, by me.

**Comment by jtrmal** [comment]
Great. Thanks a lot.
@danpovey I think ready to merge
Y.

**Pull request merged**